tests/e2e/validatecluster/validatecluster_test.go

```diff
@@ -133,7 +134,7 @@ var _ = Describe("Verify Basic Cluster Creation", Ordered, func() {
 		}, "240s", "5s").Should(ContainSubstring("test-loadbalancer"), "failed cmd: "+cmd)
 	})
 
-	It("Verifies Restart", func() {
+	/*It("Verifies Restart", func() {
```

Comment:
delete it if you want it gone, don't just comment it out

Reply:
Yes thanks